api/preprint_providers/views.py

```diff
@@ -0,0 +1,176 @@
+from rest_framework import generics
+from rest_framework import permissions as drf_permissions
+
+from modularodm import Q
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from website.models import Node, PreprintProvider
+
+from api.base import permissions as base_permissions
+from api.base.filters import ODMFilterMixin
+from api.base.views import JSONAPIBaseView
+from api.base.pagination import MaxSizePagination
+
+from api.preprint_providers.serializers import PreprintProviderSerializer
+from api.preprints.serializers import PreprintSerializer
+
+
+class PreprintProviderList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin):
+    """
+    Paginated list of verified PreprintProviders available
+
+    ##PreprintProvider Attributes
+
+    OSF Institutions have the "institutions" `type`.
+
+        name           type               description
+        =========================================================================
+        name           string             title of the preprint provider
+        id             string             unique identifier in the OSF
+        logo_path      string             a path to the preprint provider's static logo
+        banner_path    string             a path to the preprint provider's static banner
+
+    #This Request/Response
+
+    """
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.ALWAYS_PUBLIC]
+    required_write_scopes = [CoreScopes.NULL]
+    model_class = PreprintProvider
+
+    pagination_class = MaxSizePagination
+    serializer_class = PreprintProviderSerializer
+    view_category = 'preprint_providers'
+    view_name = 'preprint_providers-list'
+
+    ordering = ('name', )
+
+    def get_default_odm_query(self):
+        return Q('_id', 'ne', None) & Q('is_deleted', 'ne', True)
```

Comment:
I think `_id` should never be `None`... if so, the first part of this `Q` is superfluous